### PR TITLE
feat: add default labels to registry

### PR DESCRIPTION
**Pull request by ghost**
To specify common labels, such as an application name or an environment, just like in prom-client https://github.com/siimon/prom-client#default-labels-segmented-by-registry

### CODE REVIEW

File: src/CollectorRegistry.ts [hunk 1]
```diff
@@ -20,19 +21,38 @@ export class CollectorRegistry {
...
+          sample.labels = sample.labels || {}
+
+          if (defaultLabelNames.length > 0) {
+            // Make a copy before mutating
+            sample.labels = { ...sample.labels }
+
+            for (const labelName of defaultLabelNames) {
+              sample.labels[labelName] = sample.labels[labelName] || this.defaultLabelMap[labelName]
+            }
+          }
```
Comment (ruanyl) [inline src/CollectorRegistry.ts]: It looks to me like this piece of code can simply be:
```typescript
sample.labels = { ...this.defaultLabelMap, ...sample.labels }
```

**Comment by ruanyl** [comment]
@andrepolischuk Thanks for the PR, I left a small comment

File: src/CollectorRegistry.ts (hunk 1 above)
Comment (ghost) [reply src/CollectorRegistry.ts]: \+

**Comment by ghost** [comment]
@ruanyl I also added an example to readme

**Comment by ruanyl** [comment]
@andrepolischuk awesome, thank you! LGTM

**Comment by ghost** [comment]
@ruanyl do you plan to publish a new version?

**Comment by ruanyl** [comment]
@andrepolischuk released in `v0.2.2`

**Pull request merged**